ethcore/src/trace/db.rs

```diff
@@ -52,144 +47,74 @@ impl Key<FlatBlockTraces> for H256 {
 	}
 }
 
-/// Wrapper around `blooms::GroupPosition` so it could be
-/// uniquely identified in the database.
-#[derive(Debug, PartialEq, Eq, Hash, Clone)]
-struct TraceGroupPosition(blooms::GroupPosition);
-
-impl From<GroupPosition> for TraceGroupPosition {
-	fn from(position: GroupPosition) -> Self {
-		TraceGroupPosition(From::from(position))
-	}
-}
-
-impl HeapSizeOf for TraceGroupPosition {
-	fn heap_size_of_children(&self) -> usize {
-		0
-	}
-}
-
-/// Helper data structure created cause [u8; 6] does not implement Deref to &[u8].
-pub struct TraceGroupKey([u8; 6]);
-
-impl Deref for TraceGroupKey {
-	type Target = [u8];
-
-	fn deref(&self) -> &Self::Target {
-		&self.0
-	}
-}
-
-impl Key<blooms::BloomGroup> for TraceGroupPosition {
-	type Target = TraceGroupKey;
-
-	fn key(&self) -> Self::Target {
-		let mut result = [0u8; 6];
-		result[0] = TraceDBIndex::BloomGroups as u8;
-		result[1] = self.0.level;
-		result[2] = self.0.index as u8;
-		result[3] = (self.0.index >> 8) as u8;
-		result[4] = (self.0.index >> 16) as u8;
-		result[5] = (self.0.index >> 24) as u8;
-		TraceGroupKey(result)
-	}
-}
-
-#[derive(Debug, Hash, Eq, PartialEq)]
-enum CacheId {
-	Trace(H256),
-	Bloom(TraceGroupPosition),
-}
-
 /// Database to store transaction execution trace.
 ///
 /// Whenever a transaction is executed by EVM it's execution trace is stored
 /// in trace database. Each trace has information, which contracts have been
 /// touched, which have been created during the execution of transaction, and
 /// which calls failed.
 pub struct TraceDB<T> where T: DatabaseExtras {
-	// cache
+	/// cache
 	traces: RwLock<HashMap<H256, FlatBlockTraces>>,
-	blooms: RwLock<HashMap<TraceGroupPosition, blooms::BloomGroup>>,
-	cache_manager: RwLock<CacheManager<CacheId>>,
-	// db
-	tracesdb: Arc<KeyValueDB>,
-	// config,
-	bloom_config: BloomConfig,
-	// tracing enabled
+	cache_manager: RwLock<CacheManager<H256>>,
```

Comment:
Missing docs for `cache_manager`

Reply:
ahh... it's not a public field and no change has been made here :p but I'll add a description ;p